Fitness/Models/ProBio.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  ProBio.swift
+//  Fitness
+//
+//  Created by Cameron Hamidi on 2/28/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+struct ProBio {
+    let id: String
+    let name: String
+    let profilePic: UIImage?
+    let secondaryProfilePic: UIImage?
+    let bio: String
+    let links: [PersonalLinkObject]
+    let expertise: [String]
+    let routines: [ProRoutine]
+    let summary: String
+    
+    init(id: String, name: String, profilePic: String, secondaryProfilePic: String, bio: String, links: [PersonalLinkObject], expertise: [String], routines: [ProRoutine], summary: String) {
+        self.id = id
+        self.name = name
+        self.profilePic = UIImage(named: profilePic)
+        self.secondaryProfilePic = UIImage(named: secondaryProfilePic)
+        self.bio = bio
+        self.links = links
+        self.expertise = expertise
+        self.routines = routines
+        self.summary = summary
+    }
+    
+    public static func getPro(name: String) -> ProBio {
+        if name == "claire" {
+            return getClaire()
+        } else {
+            return getClaire()
+        }
+    }
+    
+    public static func getAllPros() -> [ProBio] {
+        return [getClaire(), getJuan(), getMadeline(), getStarks(), getCleo()]
+    }
+    
+    public static func getClaire() -> ProBio {
+        var urls: [PersonalLinkObject] = []
+        urls.append(PersonalLinkObject(url: "https://www.instagram.com/thenvironmentalistbodybuilder/", site: .instagram))
```

Comment:
Probably best to create the URL object here and pass it into the PersonalLink as a URL. This allows us to handle the cases where it fails if this data eventually comes from backend.